packages/@aws-cdk/pipelines/test/integ.newpipeline-with-cross-account-keys.ts

```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+/// !cdk-integ PipelineStack pragma:set-context:@aws-cdk/core:newStyleStackSynthesis=true
```

Comment:
This is no longer relevant when using the IntegTest construct for your tests. They always use the new style stack synthesis.

Reply:
Copy and paste error, will remove.